ext/curl/interface.c

```diff
@@ -844,6 +844,11 @@ PHP_MINIT_FUNCTION(curl)
 	REGISTER_CURL_CONSTANT(CURLPROXY_SOCKS4);
 	REGISTER_CURL_CONSTANT(CURLPROXY_SOCKS5);
 
+#if LIBCURL_VERSION_NUM >= 0x071200 /* Available since 7.18.0 */
```

Comment:
why does the comment say, available since 7.18 and the version-num const compares against 7.12?